books/free-programming-books-zh.md

```diff
@@ -536,14 +527,9 @@
   * [Ext4.1.0 中文文档](http://extjs-doc-cn.github.io/ext4api/)
 
 
-#### impress.js
-
-  * [impress.js的中文教程](https://github.com/kokdemo/impress.js-tutorial-in-Chinese)
```

Comment:
Why remove this? It works

Reply:
it's an almost empty repository, I should have deleted it in another PR I know, I forgot